view/topbar.twig

```diff
@@ -15,7 +15,7 @@
 <div id="language"><span class="navigation-font">|</span>
   {% for langcode, langdata in languages %}
   {% if request.lang != langcode %}
-  <a id="language-{{ langcode}}" class="navigation-font" href="{{ request.langurl|replace({("#{request.lang}/"): "#{langcode}/"}) }}"> {{langdata.name}}</a>
+  <a id="language-{{ langcode}}" class="navigation-font" href="{{ request.langurl|replace({("/#{request.lang}/"): "/#{langcode}/"}) }}"> {{langdata.lemma}}</a>
```

Comment:
Is it intentional that you replace `langdata.name` with `langdata.lemma` here?

Reply:
Yes, before changing it, all languages were displayed with /in_this_language**

Reply:
Interesting that when running with Docker (see pull #700), I get the same, i.e. **in_this_language**. But when I get the changes in this pull request, at least that text is fixed, and displays now "svenka" and "English" instead. So at least it seems to fix this issue for my environment.

Reply:
Fixes the text, but now I noticed that the links are not working. If I change in the URL from /fi/ to /en/, everything works. Clicking on the links doesn't seem to be working for me 👍